### PR TITLE
Go to transition after backoffice asset edition.

### PR DESCRIPTION
**Pull request by xliiv**
Added redirecting button to return-asset transition from bo bulk-edit and single edit.

### CODE REVIEW

File: src/ralph_assets/views/asset.py [hunk 1]
```diff
@@ -128,3 +128,26 @@ def handle_formset_error(self, formset_error):
...
                '"barcodes" for duplicates'))
         )
+
+    def get_success_url(self):
+        """Redirect after successfully send formset.
+
+        :return: returns URL (``str``)
+        """
+        #TODO:: clean it :P
+        from django.core.urlresolvers import reverse
```
Comment (mkurek) [inline src/ralph_assets/views/asset.py]: why import here?

Comment (xliiv) [inline src/ralph_assets/views/asset.py]: this was dirty and temp version

**Comment by andrzej-jankowski** [comment]
:ok:

**Comment by zefciu** [comment]
:ok: dokie

**Pull request merged**